docs/components/DrawerDocs.js

```diff
@@ -0,0 +1,159 @@
+const React = require('react');
+
+const { Button, Drawer } = require('mx-react-components');
+
+const Markdown = require('components/Markdown');
+
+const DrawerDocs = React.createClass({
+
+  getInitialState () {
+    return {
+      demoDrawerOpen: false,
+      currentPage: 3,
+      totalPages: 8
+    };
+  },
+
+  _handleDemoButtonClick () {
+    this.setState({
+      demoDrawerOpen: true
+    });
+  },
+
+  _handleDrawerClose () {
+    this.setState({
+      demoDrawerOpen: false
+    });
+  },
+
+  _handlePreviousClick () {
+    if (this.state.currentPage > 1) {
+      this.setState({
+        currentPage: this.state.currentPage - 1
+      });
+    }
+  },
+
+  _handleNextClick () {
+    if (this.state.currentPage < this.state.totalPages) {
+      this.setState({
+        currentPage: this.state.currentPage + 1
+      });
+    }
+  },
+
+  _renderDrawer () {
+    const styles = this.styles();
+
+    return (
+      <Drawer
+        breakPoints={{ large: 1200, medium: 1100, small: 1320 }}
+        contentStyle={styles.content}
+        navConfig={{ label: this.state.currentPage + ' of ' + this.state.totalPages, onNextClick: this._handleNextClick, onPreviousClick: this._handlePreviousClick }}
+        onClose={this._handleDrawerClose}
+        title='Demo Drawer'
+      >
+        Pellentesque finibus eros magna, ac feugiat mauris pretium posuere. Aliquam nec turpis bibendum, hendrerit eros et, interdum neque. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Nunc pulvinar tempus sollicitudin. Mauris vel suscipit dolor. Vestibulum hendrerit malesuada ipsum. Mauris feugiat dui vel leo consequat tempor. Praesent aliquet posuere consequat. Nunc vel tellus eleifend leo finibus auctor.
+      </Drawer>
+
+    );
+  },
+
+  render () {
+    return (
+      <div>
+        <h1>
+          Drawer
+          <label>A basic drawer component</label>
+        </h1>
+
+        <h3>Demo</h3>
+        <Button onClick={this._handleDemoButtonClick}>
+          Demo Drawer
+        </Button>
+        {this.state.demoDrawerOpen && this._renderDrawer()}
+
+        <h3>Usage</h3>
+        <h5>animateLeftDistance<label>Number</label></h5>
+        <p>This number represents the percent of the screen visible between the left edge of the screen and the left edge of the drawer.</p>
+
+        <h5>breakPoints<label>Object</label></h5>
+        <p></p>
```

Comment:
Here is a rough description of the breakpoints:

_When the screen size is:_
__`>=` large:__ drawer width is `maxWidth`
__`<` large and `>` medium:__ drawer width is 20% from left side
__`<=` medium:__ drawer width is 100%